Tests/SonarScanner.MSBuild.PreProcessor.Test/PreprocessorObjectFactoryTests.cs

```diff
@@ -170,6 +172,30 @@ public async Task CreateSonarWebService_WithoutOrganizationOnSonarCloud_ReturnsN
 
         server.Should().BeNull();
         logger.AssertSingleErrorExists(@"Organization parameter (/o:""<organization>"") is required and needs to be provided!");
+        logger.AssertSingleWarningExists("""
+            Did you want to connect to a local SonarQube server?
+            In version 7 of the scanner the default server changed from "http://localhost:9000" to "https://sonarcloud.io".
+            Add the parameter "/d:sonar.host.url=https://localhost:9000" to connect to your local SonarQube server.
```

Comment:
We can log:
In version 7 of the scanner, the default value for the `sonar.host.url` changed from "http://localhost:9000" to "https://sonarcloud.io". If the intention was to connect to the local SonarQube instance, please add the `"/d:sonar.host.url=https://localhost:9000"` parameter.